monitor/Makefile

```diff
@@ -0,0 +1,19 @@
+CFLAGS := $(shell python3.6m-config --cflags)
```

Comment:
same file as backend maybe use a symbolic link instead

Reply:
good idea, maybe not critical (what if the containers are physically separate?)